docs/uninstall-olm.md

```diff
@@ -0,0 +1,46 @@
+# How do I uninstall OLM?
+
+OLM is installed within a namespace called `openshift-operator-lifecycle-manager` on an OpenShift cluster. On a local minikube cluster, it is installed within the namespace `olm`. You can uninstall OLM in a symmetrical way as you did to install it. 
+
+## Uninstall Released OLM
+For uninstalling release versions of OLM, for example version 0.13.0, you can use the following command:
+
+```bash
+export olm_release=0.13.0
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
+kubectl delete ns <OLM namespace, e.g.: olm, openshift-operator-lifecycle-managment, etc>
+```
+
+To find out the version of the OLM you are using, please use the following to see the version of packageserver csv.
```

Comment:
Nit: You can identify which version of OLM you are using by inspecting the version of the packageserver `CSV`: